invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/oaipmh/search/index.js

```diff
@@ -14,7 +14,7 @@ import { NotificationController } from "@js/invenio_administration/src/ui_messag
 const domContainer = document.getElementById("invenio-search-config");
 
 const sortColumns = (columns) =>
-  Object.entries(columns).sort((a, b) => a[1].order > b[1].order);
+  Object.entries(columns).sort((a, b) => a[1].order - b[1].order);
```

Comment:
why was this changed to `-`?

Reply:
Because otherwise it doesn't work, we had the same issue in [`invenio-adminsitration`](https://github.com/inveniosoftware/invenio-administration/blob/main/invenio_administration/assets/semantic-ui/js/invenio_administration/src/search/search.js#L29), we updated it there but forgot to change it here.
This is how it was looking before:
![image](https://user-images.githubusercontent.com/15194802/191766465-12475e5d-113a-4287-8c37-2e988e000dcb.png)

Reply:
cross-checked with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort, good to go